Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:16.04
```

Comment:
I see this is meant to be https://github.com/aeternity/epoch/blob/b7f9e78f5fc31df179d7773502fbede92b235bd2/docker/Dockerfile-builder